daff/README.md

```diff
@@ -0,0 +1,228 @@
+# Directory Contents
+
+**diff2/** - This folder contains html diff2 table output of `diff2.js` script.
+
+**diff3/** - This folder contains html diff3 table output of `diff3.js` script.
+
+**mergedData/** - This folder contains updated local file after having diff changes merged in. It is an output of `diff3.js`.
+
+**data/** - This folder contains input files for diff scripts, metadata of input files and results of runinng test cases.
+
+**calcColumnSimilarity.js** - This script completes preproccessing steps for `diff2.js`/`diff3.js` and produces meta data
```

Comment:
Please add another subdirectory in `data/` directory to include all input files and the output file needed for running `calcColumnSimilarity.js`. Thanks.